### ISSUE TITLE
Hello, I'm sorry to disturb you again. I would like to ask you if you choose this fcos as the one-stage model. Is that because what features of this model can be applied to semi supervision? Or can the basic model be replaced with another stage model?

### ISSUE THREAD

**Issue by mary-0830** [facebookresearch/unbiased-teacher-v2#3]
Hello, I'm sorry to disturb you again. I would like to ask you if you choose this fcos as the one-stage model. Is that because what features of this model can be applied to semi supervision? Or can the basic model be replaced with another stage model?

**Comment by mary-0830** [comment]
Can the Yolo series model be used?

**Comment by ycliu93** [comment]
We used the FCOS as the one-stage object detector due to its popularity in the community. There are some other reasons we did so: it does not need to tune some hyper-parameters of anchors (like the number of anchors or pre-defined sizes), so it is easier to adapt to other object detection datasets with potentially higher performance. 

We were thinking of extending it to YOLO series object detectors, but the complexities and heterogeneity of different object detectors made us use the FCOS (single-stage anchor-free) and Faster-RCNN (two-stage anchor-based) at this stage.

I am happy to provide some assistance or collaborations if you are interested in applying it to YOLO series models.
